pandas/tests/frame/test_analytics.py

```diff
@@ -1442,6 +1442,26 @@ def test_any_datetime(self):
         expected = Series([True, True, True, False])
         tm.assert_series_equal(result, expected)
 
+    def test_any_all_bool_only(self):
+
```

Comment:
is this near the other original test?

Reply:
Yes, it is directly below the test in the previous commit that introduced this bug.